plugin.video.simpsonsworld/resources/lib/globals.py

```diff
@@ -0,0 +1,198 @@
+import sys, os
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+import urllib, urllib2
+import json
+import base64
+from adobepass.adobe import ADOBE
+
+
+
+addon_handle = int(sys.argv[1])
+ADDON = xbmcaddon.Addon(id='plugin.video.simpsonsworld')
```

Comment:
No need for the id here. Passing the id to Addon() is only needed if you want to access another plugin from yours. xbmcaddon.Addon() is enough here